everware/home_handler.py

```diff
@@ -128,7 +128,11 @@ def get(self):
             ))
             return
 
-        stat = yield user.spawner.poll()
+        try:
+            stat = yield user.spawner.poll()
+        except Exception as e:
+            self.log.warn("Error polling user(%s) conainer state" % repr(user))
```

Comment:
you don't need to call `repr()` there is a formatting code to get `repr()` instead of `str()`